docs/content/latest/secure/enable-authentication/ysql_hba_conf-configuration.md

```diff
@@ -21,21 +21,25 @@ showAsideToc: true
   </li>
 </ul>
 
-The client authentication in YugabyteDB is managed by the T-Server <code>[--ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv) </code>
-configuration flag, which works similar to the <code>pg_hba.conf</code> file in PostgreSQL. 
+Client authentication in YugabyteDB is managed by the YB-TServer [`--ysql_hba_conf_csv`](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv) configuration flag, which works similar to the `pg_hba.conf` file in PostgreSQL.
```

Comment:
which works similar to the `pg_hba.conf` file in PostgreSQL. --> which works similarly to the `pg_hba.conf` file in PostgreSQL.